### ISSUE TITLE
5.1.0 - iOS pod install issue

### ISSUE THREAD

**Issue by cristianoccazinsp** [Agontuk/react-native-geolocation-service#223]
Getting the following after the 5.1.0 upgrade:

```
Installing react-native-geolocation-service 5.1.0 (was 5.0.0)
[!] The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `react-native-geolocation-service` depends upon `React-Core`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
```

**Comment by ljcp** [comment]
same issue here, I reverted back to 5.0.0

**Comment by Agontuk** [comment]
Fixed in v5.1.1

**Comment by helloeyesimple** [comment]
5.3.0 still having the same problem
